src/jira.js

```diff
@@ -1086,4 +1109,36 @@ export default class JiraApi {
       pathname: '/status',
     })));
   }
+
+  /** Get a Dev-Status summary by issue ID
+   * @name getDevStatusSummary
+   * @function
+   * @param {string} issueId - id of issue to get
+   */
+  getDevStatusSummary(issueId) {
+    return this.doRequest(this.makeRequestHeader(this.makeDevStatusUri({
+      pathname: '/summary',
+      query: {
+        issueId,
+      },
+    })));
+  }
+
+  /** Get a Dev-Status detail by issue ID
+   * @name getDevStatusDetail
+   * @function
+   * @param {string} issueId - id of issue to get
+   * @param {string} applicationType - type of application (stash, bitbucket)
+   * @param {string} dataType - info to return (repository, pullrequest)
+   */
+  getDevStatusDetail(issueId, applicationType, dataType) {
+    return this.doRequest(this.makeRequestHeader(this.makeDevStatusUri({
+      pathname: '/detail',
+      query: {
+        issueId,
+        applicationType,
+        dataType,
```

Comment:
From what I have seen (I tried calling the API in different ways) all of those are required and are very specific to what you want to grab. For instance dataType returns different information about that issue in the application. But an issue could have multiple applicationTypes (bitbucket, github) at the same time.

Here is the link to the JIRA issue about this https://jira.atlassian.com/browse/JRACLOUD-35331